cli/commands/stack/action.go

```diff
@@ -34,8 +36,31 @@ func RunGenerate(ctx context.Context, opts *options.TerragruntOptions) error {
 	return generateStack(ctx, opts)
 }
 
+// Run execute stack command.
+func Run(ctx context.Context, opts *options.TerragruntOptions) error {
+	stacksEnabled := opts.Experiments[experiment.Stacks]
+	if !stacksEnabled.Enabled {
+		return errors.New("stacks experiment is not enabled use --experiment stacks to enable it")
+	}
+
+	if err := RunGenerate(ctx, opts); err != nil {
+		return err
+	}
+
+	// prepare options for execution
+	// navigate to stack directory
+	opts.WorkingDir = filepath.Join(opts.WorkingDir, stackDir)
+	// remove 0 element from args
+	opts.TerraformCliArgs = opts.TerraformCliArgs[1:]
+	opts.TerraformCommand = opts.TerraformCliArgs[0]
+	opts.OriginalTerraformCommand = strings.Join(opts.TerraformCliArgs, " ")
+
+	return runall.Run(ctx, opts)
+}
+
 func generateStack(ctx context.Context, opts *options.TerragruntOptions) error {
 	opts.TerragruntStackConfigPath = filepath.Join(opts.WorkingDir, defaultStackFile)
+	opts.Logger.Infof("Generating stack from %s", opts.TerragruntStackConfigPath)
```

Comment:
Does this need to be in info?

Reply:
Without an info message, it is not clear what is happening, the user doesn't see any message and it is not clear which units got processed